src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -202,10 +205,56 @@ public void actionbar(Player... players) {
      * @return This {@link JSONMessage} instance
      */
     public JSONMessage color(ChatColor color) {
+        if (!color.isColor())
+            throw new IllegalArgumentException(color.name() + " is not a color.");
+        last().setColor(color.name().toLowerCase());
```

Comment:
I consider changing this to `return color(color.name.toLowerCase());` to just have one major method that handles this instead of several that all do the same (`last().setColor(...)`)

Reply:
it's pretty simple either way, but for best practices sake, yeah that would be a good change :)